src/transformState.js

```diff
@@ -5,7 +5,24 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const { type, extraData, keysToRemove } of actions) {
+    switch (type) {
+      case 'addProperties':
+        Object.assign(state, extraData);
+        break;
+
+      case 'removeProperties':
+        keysToRemove.forEach(key => {
+          delete state[key];
+        });
+        break;
+
+      default:
+        Object.keys(state).forEach(key => {
+          delete state[key];
+        });
```

Comment:
you need to handle clear case as well (I understand it's handled in default, but default shouldn't be used for such reasons. In default you should throw an error that there's no such method or something like this)

Reply:
ok sir